lib/src/model/model_element.dart

```diff
@@ -791,3 +786,16 @@ abstract class ModelElement
 
   String get linkedObjectType => _packageGraph.dartCoreObject;
 }
+
+extension on ElementAnnotation {
+  bool get isVisibleAnnotation {
```

Comment:
What does the word "visible" mean here?
When I look at the implementation, I see "pragma".

Reply:
> What does the word "visible" mean here?

It means visible (or present) in the output HTML.

> When I look at the implementation, I see "pragma".

Yeah, not obvious, on line 379 we have a doc comment for ModelElement.annotations that says:

> Does not include annotations with `null` elements or that are otherwise supposed to be invisible (like `@pragma`). 

I could duplicate that text, or maybe just move it to be doc comment on this getter? I'd be fine with either; I'm good with more documentation.

Reply:
I duplicated the text.